spring-integration-core/src/main/java/org/springframework/integration/config/xml/PointToPointChannelParser.java

```diff
@@ -78,6 +78,7 @@ else if ((queueElement = DomUtils.getChildElementByTagName(element, "rendezvous-
 						"The 'fixed-subscriber' attribute is not allowed when a <queue/> child element is present.",
 						element);
 			}
+			Assert.state(builder != null, "builder must not be null");
```

Comment:
Why not throw our friend "dummy" exception after error instead?
I mean this extra call is redundant and misleading.